hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -437,36 +437,36 @@ spec = around showLogsOnFailure $
             void $ writeFileTextEnvelope hydraSK Nothing hydraSKey
             void $ writeFileTextEnvelope cardanoSK Nothing cardanoSKey
 
-            -- generate node state to save to a file
-            openState :: OpenState Tx <- generate arbitrary
-            let headParameters = parameters openState
-                stateContestationPeriod = Hydra.Chain.contestationPeriod headParameters
-
-            -- generate one value for CP different from what we have in the state
-            differentContestationPeriod <- generate $ arbitrary `suchThat` (/= stateContestationPeriod)
-
-            -- alter the state to trigger the errors
-            let alteredNodeState =
-                  Open
-                    ( openState
-                        { parameters =
-                            headParameters{contestationPeriod = differentContestationPeriod, parties = []}
-                        }
-                    )
-
-            -- save altered node state
+            let alice = deriveParty hydraSKey
```

Comment:
Could re-write this test:
1. Start the node with one configuration and persistency
2. Initialize a head
3. Start the node with a different configuration and same persistency